Framework/include/PrepNTupleVars.h

```diff
@@ -358,13 +354,12 @@ class PrepNTupleVars
         auto& newJetsAK8_NsubjettinessTau2  = tr.createDerivedVec<float>("JetsAK8"+name+"_NsubjettinessTau2", jc.JetsAK8.size());
         auto& newJetsAK8_NsubjettinessTau3  = tr.createDerivedVec<float>("JetsAK8"+name+"_NsubjettinessTau3", jc.JetsAK8.size());
         auto& newJetsAK8_softDropMass       = tr.createDerivedVec<float>("JetsAK8"+name+"_softDropMass", jc.JetsAK8.size());
-        auto& newJetsAK8_prunedMass         = tr.createDerivedVec<float>("JetsAK8"+name+"_prunedMass", jc.JetsAK8.size());
```

Comment:
Does the top tag not need the "prunedMass" variable?

Reply:
I could only find reference to it in `PrepNTupleVars.h`, no where to be seen in `RunTopTagger` or `SetupTopTagger`. Unless it is somehow implicitly passed.